AENS.md

```diff
@@ -66,19 +65,47 @@ could allow us an easier path for future update to the fee structures.
 
 Every entry in the `.test` namespace pays the same amount of fees.
 
+The new mechanism planted in Lima hard fork introduces auctions.
+
+The claim action becomes claim attempt. For short names claim transaction
+doesn't set ownership of the name. It can be followed by another
+claim from an account different than one set in preclaim for given name.
+Names below 32 characters are considered short.
+
+There is finite amount of time when the follow up claim is allowed.
```

Comment:
Why is there a waiting time between claim transactions, when anyone is allowed to send them? Or do you mean claim by the same party?

Reply:
I think my description is still not clear. 

Claim by different parties.

 Scenario:
1. Pre-claim by X
2. Claim by X at H1 ( T = timout(length(name)) and set bid timeout)
3. Claim by Y at H2 and (H2-H1) < T
4. Claim by X at H3 and (H3-H2) <T
5. ... more than T blocks passes
6. name claimed by X at H4 (H4=H3+T)

If in number 4 X bid at H3' and H3' > T then it would be invalid transaction

Reply:
The idea is to have classic english type auction. So, we need time frame when next bid is valid.